status_test.go

```diff
@@ -61,3 +61,30 @@ func TestStatusList(t *testing.T) {
 		t.Fatal("Incorrect entry path: ", entry.IndexToWorkdir.NewFile.Path)
 	}
 }
+
+func TestStatusNothing(t *testing.T) {
+	t.Parallel()
+	repo := createTestRepo(t)
+	defer cleanupTestRepo(t, repo)
+
+	seedTestRepo(t, repo)
+
+	opts := &StatusOptions{}
+	opts.Show = StatusShowIndexAndWorkdir
+	opts.Flags = StatusOptIncludeUntracked | StatusOptRenamesHeadToIndex | StatusOptSortCaseSensitively
+
+	statusList, err := repo.StatusList(opts)
+	checkFatal(t, err)
+
+	entryCount, err := statusList.EntryCount()
+	checkFatal(t, err)
+
+	if entryCount != 0 {
+		t.Error("expected no statuses in empty repo")
```

Comment:
nit:

```suggestion
		t.Fatal("expected no statuses in empty repo")
```

if this fails, then the preconditions for the rest of the test case are not met, which will make the rest of the assertions fail, causing log spam.